src/trace_processor/tables/profiler_tables.py

```diff
@@ -581,6 +581,47 @@
                 deobfuscation mapping was provided for it.'''
         }))
 
+HPROF_STATS_TABLE = Table(
```

Comment:
Do we need a new table for this? Feels like it could just be stats in the stats table?